packages/visualizations/src/components/Table/types.ts

```diff
@@ -30,7 +34,10 @@ export type NumberColumn = BaseColumn & {
      *
      * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options
      */
-    options?: Intl.NumberFormatOptions;
+    options?: {
+        display?: (v: string) => string;
+        intl?: Intl.NumberFormatOptions;
```

Comment:
Please move the comment about `NumberFormat` options just above the intl property